changelog/Microsoft.DirectoryServices.json

```
@@ -7431,6 +7431,24 @@
       "WorkloadArea": "Identity and access",
       "SubArea": ""
     },
+    {
+      "ChangeList": [
+        {
+          "Id": "db8cde72-c86c-4a67-9c0f-65663c90d0df",
+          "ApiChange": "Property",
+          "ChangedApiName": "isMultiValued",
+          "ChangeType": "Addition",
+          "Description": "Added the `isMultiValued` property to the [extensionProperty](https://learn.microsoft.com/en-us/graph/api/resources/extensionProperty?view=graph-rest-beta) resource.",
```

Suggestion to enhance the description to improve the discoverability of this capability. Dan has called out this capability for directory extensions and coming soon for schema extensions.
```suggestion
          "Description": "Added the `isMultiValued` property to the [extensionProperty](https://learn.microsoft.com/en-us/graph/api/resources/extensionProperty?view=graph-rest-beta) resource. You can now store a collection of values in a directory extension property.",
```